api/templates/client.mustache

```diff
@@ -294,6 +294,11 @@ func (c *APIClient) prepareRequest(
 		return nil, err
 	}
 
+	// Prepend request base path, if applicable
+    if c.cfg.Path != "" {
+        url.Path = c.cfg.Path + url.Path
```

Comment:
Better to handle `/` properly. I don't know if there is an existing utility for it. But users can always self add/remove the `/` on demand.